doc/theory.md

```diff
@@ -52,3 +52,18 @@ Finally, FLOSS diffs the emulator memory states from before and
   6. Emulate decoder functions using extracted arguments and emulator state
   7. Diff memory state from before and after decoder emulation
   8. Extract human-readable strings from memory state difference
+
+## Go String Extraction
```

Comment:
how about moving this to a separate MD file as well? we could introduce language specific extraction in general there and then talk about the different languages